src/openforms/authentication/tests/test_migrations.py

```diff
@@ -0,0 +1,34 @@
+from openforms.utils.tests.test_migrations import TestMigrations
+
+from ..constants import AuthAttribute
+
+
+class MigrateAuthInfoDataForwardTest(TestMigrations):
+    migrate_from = "0001_initial"
+    migrate_to = "0002_migrate_submission_data"
+    app = "of_authentication"
+
+    def setUpBeforeMigration(self, apps):
+        Submission = apps.get_model("submissions", "Submission")
+        Form = apps.get_model("forms", "Form")
+
+        form = Form.objects.create(name="Form", slug="form")
+        submission = Submission.objects.create(
+            form=form,
+            kvk="123456789",
+            bsn="",
+            pseudo="",
+            auth_plugin="test-kvk-plugin",
+            auth_attributes_hashed=True,
```

Comment:
that doesn't make sense with the unhashed values - can you add some explicit checks for submissions with/without hashed details? Keep in mind that hashing a submission also hashes the _empty_ fields!